wpilibjExamples/src/main/java/edu/wpi/first/wpilibj/examples/solenoid/Robot.java

```diff
@@ -25,34 +37,134 @@ public class Robot extends TimedRobot {
   private final Joystick m_stick = new Joystick(0);
 
   // Solenoid corresponds to a single solenoid.
+  // In this case, it's connected to channel 0 of a PCM with the default CAN ID.
   private final Solenoid m_solenoid = new Solenoid(PneumaticsModuleType.CTREPCM, 0);
 
   // DoubleSolenoid corresponds to a double solenoid.
+  // In this case, it's connected to channels 1 and 2 of a PH with the default CAN ID.
   private final DoubleSolenoid m_doubleSolenoid =
-      new DoubleSolenoid(PneumaticsModuleType.CTREPCM, 1, 2);
+      new DoubleSolenoid(PneumaticsModuleType.REVPH, 1, 2);
+
+  // Compressor connected to a PH with a default CAN ID
+  private final Compressor m_compressor = new Compressor(PneumaticsModuleType.REVPH);
+
+  // External analog pressure sensor
```

Comment:
I know this is for an RLI, but it doesn't really make sense for an analog pressure sensor to be used when you're using the REV PH, since it has an input for the analog pressure sensor.

Reply:
What do you suggest?

Reply:
Don't try to do too much in an example just to RLI it. I'm supportive of the discussion in #5005